.env.sample

```diff
@@ -4,11 +4,11 @@
 #
 # Please use `openssl rand -hex 32` to create SECRET_KEY
 
-DATABASE_URL=postgres://user:pass@postgres:5432/outline
-DATABASE_URL_TEST=postgres://user:pass@postgres:5432/outline-test
-SECRET_KEY=F0E5AD933D7F6FD8F4DBB3E038C501C052DC0593C686D21ACB30AE205D2F634B
+DATABASE_URL=postgres://user:pass@localhost/outline
```

Comment:
We should leave the port in the example here

Reply:
Fixed